cpu/nrf5x_common/periph/gpio.c

```diff
@@ -180,7 +189,11 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
     /* configure pin as input */
     gpio_init(pin, mode);
     /* set interrupt priority and enable global GPIOTE interrupt */
+#ifdef CPU_FAM_NRF9160
+    NVIC_EnableIRQ(GPIOTE0_IRQn);
+#else
```

Comment:
with the added compat define we can drop this

Reply:
Done.